### ISSUE TITLE
Request for enhancement in `selector-no-type` to allow child, but not descendant, combinator

### ISSUE THREAD

**Issue by mik01aj** [stylelint/stylelint#1941]
Here's what I'm trying to do:

    .className > label {} // I want to allow this...
    .className label {}   // ...but not this

My current config for the [`selector-no-type`](https://github.com/stylelint/stylelint/tree/master/src/rules/selector-no-type) rule:

```json
        "selector-no-type": [true, {
            ignore: ["compounded", "descendant"],
        }],
```

With this config, both lines are fine. When I remove `descendant`, both are flagged as errors.

A possible solution would be to introduce a new `child` value for the `ignore` list. But that would be a braking change.

I'm using stylelint `7.3.1`.

**Comment by ntwb** [comment]
Thanks for creating this issue but we are closing it as issues need to follow our issue template, so that we can clearly understand your particular circumstances.

Please help us to help you by [recreating the issue](https://github.com/stylelint/stylelint/issues/new) using the template.

**Comment by evilebottnawi** [comment]
@mik01aj Reopen your issue after use template

**Comment by evilebottnawi** [comment]
@ntwb i think we should standardize response about not use template. You may want to connect robot that will do it for us?

**Comment by mik01aj** [comment]
@ntwb I went through the template and made sure that I included all the required information. Do you actually need the copypasted template text?

**Comment by mik01aj** [comment]
nevermind, I submitted #1942
